psalm-baseline.xml

```diff
@@ -2621,8 +2578,42 @@
     <PossiblyFalseArgument>
       <code><![CDATA[$simpleXml->asXML()]]></code>
     </PossiblyFalseArgument>
+    <PossiblyNullArgument>
+      <code>$entityListenersXml</code>
+      <code>$optionsXml</code>
+    </PossiblyNullArgument>
     <PossiblyNullReference>
       <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addAttribute</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
+      <code>addChild</code>
```

Comment:
This block should go away once https://github.com/vimeo/psalm/pull/9745 is merged and released.